src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Cart/index.html.twig

```diff
@@ -24,3 +24,20 @@
  *#}
 
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
+
+{% block content %}
+    {% block carts_listing %}
+      <div class="row">
+        <div class="col-12">
+          {% include '@PrestaShop/Admin/Common/Grid/grid_panel.html.twig' with {'grid': cartGrid} %}
+        </div>
+      </div>
+    {% endblock %}
+{% endblock %}
+
+{% block javascripts %}
+  {{ parent() }}
+
+  <script src="{{ asset('themes/default/js/bundle/pagination.js') }}"></script>
```

Comment:
we shouldn't mix default theme and new theme :thinking:

Reply:
Oh indeed I didn't see it And I doubt this will work with the new grid, maybe it could simply be removed

Reply:
Ok... so apparently we don't have the equivalent in the new theme And it is used this way in ALL migrated grids We should refactor this for the new theme but not in this PR